src/test/java/IMDbTop250Scraper.java

```diff
@@ -0,0 +1,92 @@
+import com.codeborne.selenide.*;
+ import java.util.ArrayList;
+import java.util.List;
+import static com.codeborne.selenide.Selenide.*;
+
+public class IMDbTop250Scraper {
+
+    public static void main(String[] args) {
+        configureSelenide();
+        openImdbPage();
+        openNavigationMenu();
+        navigateToTop250();
+        List<String> movieData = scrapeMovies(20);
+        printTopMovies(movieData);
+        double averageRating = calculateAverageRating(movieData);
+        printAverageRating(averageRating);
+    }
+
+    private static void configureSelenide() {
+        Configuration.timeout = 10000;
+    }
+    private static void openImdbPage() {
+        open("https://www.imdb.com");
+    }
+    private static void openNavigationMenu() {
+        $("#imdbHeader-navDrawerOpen").should(Condition.exist).click();
+    }
+    private static void navigateToTop250() {
+        SelenideElement top250Link = $("aside.drawer a:nth-child(2)");
+        top250Link.should(Condition.exist).click();
+    }
+
+    private static List<String> scrapeMovies(int limit) {
```

Comment:
I changed the drescription of the task, as it is quite tough and time consuming, when I was creating this task I thought  we will have more time to dig into it. So I removed some assertions.

Task:
Go to the page
Get 5 first movies from the list and print them with avg rating.

`
ElementsCollection movieList= $$(".ipc-metadata-list-summary-item");

int count = 0; //can be constant
for (SelenideElement movie : movieList) {
if (count >= 5) break;

//todo: here some action

count++;

`